phpBB/includes/ucp/ucp_register.php

```diff
@@ -257,6 +270,19 @@ function main($id, $mode)
 					$error[] = $user->lang['NEW_PASSWORD_ERROR'];
 				}
 			}
+			/**
+			* Check UCP registration data after they are submitted
+			*
+			* @event core.ucp_register_data_after
+			* @var	bool	submit		Do we display the form only
+			*							or did the user press submit
+			* @var	array 	data		Array with current ucp registration data
+			* @var	array	cp_data		Array with custom profile fields data
+			* @var	array 	error		Array with list of errors
+			* @since 3.1.4-RC1
+			*/
+			$vars = array('submit', 'data', 'cp_data', 'error');
```

Comment:
I guess nobody every payed a close attention to what the correct approach for this would be. Could you do me the favor and create ticket for this on the tracker?